tests/ui/resolve/resolve-bad-visibility.stderr

```diff
@@ -16,7 +16,7 @@ error[E0742]: visibilities can only be restricted to ancestor modules
 LL | pub(in std::vec) struct F;
    |        ^^^^^^^^
 
-error[E0433]: failed to resolve: use of unresolved module or unlinked crate `nonexistent`
+error[E0433]: cannot find item `nonexistent` in this scope
```

Comment:
```suggestion
error[E0433]: cannot find item `nonexistent` in the crate root
```
This is 2015 edition, so imports and visibilities are resolved in the root module.